pkg/k8s/client/cell.go

```diff
@@ -409,7 +409,7 @@ func isConnReady(c kubernetes.Interface) error {
 	return err
 }
 
-func toVersionInfo(rawVersion string) *versionapi.Info {
+func ToVersionInfo(rawVersion string) *versionapi.Info {
```

Comment:
Nit. This seems to be only used by the fake client, so it could be directly moved there.

Reply:
nice find. fixed.